.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  arrowParens: 'avoid',
+  bracketSameLine: true,
+  bracketSpacing: false,
+  singleQuote: true,
+  trailingComma: 'all',
+};
```

Comment:
We don't have a semantic style for code we use right now, let's remove this.

Reply:
It could stay for now and we discuss it at a meeting later on it. It'd be nice if each project was at least consistent in itself.